### PR TITLE
Add a Gitter chat badge to README.md

### PR DESCRIPTION
**Pull request by gitter-badger**
### kframework/k now has a Chat Room on Gitter

@cos has just created a chat room. You can visit it here: [https://gitter.im/kframework/k](https://gitter.im/kframework/k?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&content=body_link).

This pull-request adds this badge to your README.md:

[Gitter](https://gitter.im/kframework/k?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=body_badge)

If my aim is a little off, please [let me know](https://github.com/gitterHQ/readme-badger/issues).

Happy chatting.

PS: [Click here](https://gitter.im/settings/badger/opt-out) if you would prefer not to receive automatic pull-requests from Gitter in future.

### CODE REVIEW

**Comment by cos** [comment]
don't merge this for now

**Comment by cos** [comment]
we're still testing chat clients

File: README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ This is a readme file for the developers.
 
 # Prerequisites
 
+[![Join the chat at https://gitter.im/kframework/k](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/kframework/k?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
```
Comment (grosu) [inline README.md:7]: got it.

**Comment by cos** [comment]
we'll add the badge it in a different place

**Pull request closed without merging**